From dc476081db6f76458b9fe78a1ba70505c8fa1e4f Mon Sep 17 00:00:00 2001
From: Frantisek Sumsal <frantisek@sumsal.cz>
Date: Wed, 24 Jan 2024 19:19:29 +0100
Subject: [PATCH] test: use the default nsec3-iterations value

In Knot 3.2 the nsec3-iterations default was changed to 0 and Knot now
issues a warning if the value is > 0. Let's just use the default value,
since it's not something that's important for our tests.

(cherry picked from commit 0652cf8e7b08c97a52a0995eb8f0dc6bb20a4de0)

Related: RHEL-1086
---
 test/knot-data/knot.conf | 1 -
 1 file changed, 1 deletion(-)

diff --git a/test/knot-data/knot.conf b/test/knot-data/knot.conf
index 6ea0cca3db..cfe478fe1c 100644
--- a/test/knot-data/knot.conf
+++ b/test/knot-data/knot.conf
@@ -52,7 +52,6 @@ policy:
       ksk-lifetime: 365d
       ksk-submission: parent_zone_sbm
       nsec3: on
-      nsec3-iterations: 10
       propagation-delay: 1s
       signing-threads: 4
       zone-max-ttl: 1s
